### DISCUSSION TITLE
Deploying | ShellHub

### DISCUSSION THREAD

**Discussion by giscus** [shellhub-io/shellhub#4126] [General]
# Deploying | ShellHub

Learn how to deploy a self-hosted ShellHub instance on your own server.

https://docs.shellhub.io/self-hosted/deploying

**Comment by Florian-Laluque** [comment]
Hello, 

For the record, if you want to use Caddy as a reverse proxy, it will work but you will have to force some headers, the X-Real-IP to send the ip of the real client, and the X-Forwarded-Port. All other headers will be ok by default.

Exemple : 

shellhub.yourdomain.com {

	reverse_proxy * http://backend:80 {
	    header_up X-Real-IP {remote_host}
        header_up X-Forwarded-Port 443
	}
}
